persistence/gov.go

```diff
@@ -7,18 +7,23 @@ import (
 	"strconv"
 
 	"github.com/pokt-network/pocket/persistence/types"
-	"github.com/pokt-network/pocket/shared/modules"
+)
+
+// TODO : Deprecate these two constants when we change the persistenceRWContext interface to pass the `paramName`
```

Comment:
Can you create a github issue (it can be a starter task for someone) to change the interface for this, preferably with an example.

Reply:
https://github.com/pokt-network/pocket/issues/286